### ISSUE TITLE
Broken bookmark URL blocks page loading

### ISSUE THREAD

**Issue by apiraino** [nextcloud/bookmarks#642]
### Steps to reproduce
1. Saved a new bookmark from Feedly
2. Bookmark URL was saved broken
3. Webapp page does not load anymore (see attached JS error)
4. Android Bookmark app works fine

### Expected behaviour
A broken URL should not block the entire Bookmark webapp loading

### Actual behaviour
The broken URL blocks indeed the entire Bookmark webapp

### Server configuration
**PHP version:**
7.0.2

**Nextcloud version:** (see Nextcloud admin page)
1.5.0.2

**Bookmarks version:** (see Nextcloud apps page)
0.16.0

**Updated from an older Nextcloud or fresh install:**
fresh install

### Client configuration
Firefox 65.0b12

**Operating system:**
Ubuntu 18.10

#### Browser log
```
javascript error from browser console:

TypeError: http://hacking of artificial intelligence is an emerging security crisis https/www.wired.co.uk/article/artificial-intelligence-hacking-machine-learning-adversarial is not a valid URL. main.bundle.js:21194:28
	getColor https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:21194
	onRender https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:34437
	triggerMethod https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:21526
	1 https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:22730
	render https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:23571
	render self-hosted:972
	renderView https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:22788
	addChildView https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:24379
	_addChild https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:24349
	_addChild self-hosted:978
	forEach https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:2150
	_showCollection https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:24197
	_renderChildren https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:24172
	render https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:23997
	render self-hosted:976
	triggerEvents https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:440
	triggerApi https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:426
	eventsApi https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:225
	trigger https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:416
	reset https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:1022
	success https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:1120
	fire https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:6858
	fireWith https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:6988
	done https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:12895
	callback https://mydomain.com/apps/bookmarks/js/dist/main.bundle.js?v=54c883df-15:13138
```

Thank you!

**Comment by apiraino** [comment]
great work @marcelklehr  thank you! :+1:

**Comment by apiraino** [comment]
Tested the fix on 0.16.1, works as expected

Thanks again
